certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "firstName": {
+                  "type": "string",
+                  "description": "firstName"
+                },
+                "lastName": {
+                  "type": "string",
+                  "description": "lastName"
+                },
+                "email": {
+                  "type": "string",
+                  "description": "email",
+                  "title": "",
+                  "x-ms-visibility": "Advanced"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Individuals/{id}/Activities": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Add Activity",
+        "description": "Add Activity",
+        "operationId": "Add-Activity",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Individual ID",
+            "x-ms-summary": "Individual ID"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "category": {
+                  "type": "string",
+                  "description": "category"
+                },
+                "text": {
+                  "type": "string",
+                  "description": "text"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url"
+                },
+                "activityDate": {
+                  "type": "string",
+                  "description": "activityDate"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/Individuals/Profile/{idOrRecordNumberOrEmail}/{pageNumber}/": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.